### PR TITLE
Fix markup liveblog

### PR DESCRIPTION
**Pull request by kul3r4**
Creates metadata from liveBlogUpdate array from the server.
@sebastianbenz PTAL
@jkingyens FYI: This would fix the array problem that was making the metadata unvalid. I also added some of your suggestions from #398

### CODE REVIEW

File: backend/amp-live-list.go [hunk 1]
```diff
@@ -119,20 +149,38 @@ func readStatus(r *http.Request) int {
...
+}
+
+func createPage(newStatus int, timestamp time.Time, r *http.Request) Page {
 	if newStatus > len(blogs) {
 		newStatus = len(blogs)
 	}
 	blogItems := getBlogEntries(newStatus, timestamp)
 	score := createScore(newStatus, 0)
-	return Page{BlogItems: blogItems, FootballScore: score}
+	metadata, _ := json.Marshal(createMetadata(r))
```
Comment (sebastianbenz) [inline backend/amp-live-list.go]: I think you can simply pass the json directly to the template.

File: backend/amp-live-list.go [hunk 2]
```diff
@@ -119,20 +149,38 @@ func readStatus(r *http.Request) int {
...
+	return Page{BlogItems: blogItems, FootballScore: score, BlogMetadata: string(metadata)}
 }
 
 func renderSample(w http.ResponseWriter, r *http.Request, filePath string) {
-	t, _ := template.ParseFiles(filePath)
+	t, error := template.ParseFiles(filePath)
+	log.Println(error)
```
Comment (sebastianbenz) [inline backend/amp-live-list.go]: correctly handle the error, `panic(error)` is OK if you parse the template on startup -> don't parse the files with every request, instead cache the parsed template.

File: src/50_Samples_%26_Templates/Live_Blog.html [hunk 3]
```diff
@@ -52,20 +53,7 @@
...
-              "url": "<%host%>/samples_templates/live_blog/#{{.ID}}",
-              "datePublished": "{{.MetadataTimestamp}}",
-              "articleBody": {
-                "@value": "{{.Text}}",
-                "@type": "rdf:HTML"
-              }
-            }
-          ]
-          {{end}}
+          "liveBlogUpdate": {{.BlogMetadata}}
```
Comment (sebastianbenz) [inline src/50_Samples_%26_Templates/Live_Blog.html]: nice! I was afraid you'd needed to convert the whole json.

**Comment by jkingyens** [comment]
@kul3r4 Thank you for the metadata fix. But with regards to formatting suggestions in #398 but please keep PRs separate in the future.

**Pull request merged**